### PR TITLE
Generating mock data

### PR DESCRIPTION
**Pull request by clfhhc**
[https://trello.com/c/QZikuxU8/2-setup-databases-m](url)

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -19,6 +19,27 @@ class MySqlDatabase {
...
+
+const truncateAllTables = () => connection.query(`SET FOREIGN_KEY_CHECKS = 0;
+  TRUNCATE users;
+  TRUNCATE rooms;
+  TRUNCATE reviews;
+  SET FOREIGN_KEY_CHECKS = 1;`)
+  .then(results => JSON.parse(JSON.stringify(results)));
 
+// const queryReviewsByRoomId = (room_id, queryObj) => {
+
```
Comment (tonaramos) [inline database/index.js]: Use /** ... */ for multi-line comments. - style guide.

File: database/insertMockData.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+const db = require('./index');
+
+
+const numPics = 348;
+const templateUrl = 'https://s3-us-west-1.amazonaws.com/nappbnbreviews/portait**.jpeg';
+
+const numUsers = randomGenerator.generateInteger(1000, 100);
+const userObjs = [];
+
+for (let i = 0; i < numUsers; i++) {
```
Comment (tonaramos) [inline database/insertMockData.js]: You could use the js higher-order function forEach instead  of a for loop.

File: database/insertMockData.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+const addRates = (j, key) => {
+  reviewRates[key] = randomGenerator.generateInteger(5, 3);
+  avgReviewRates[key] = ((avgReviewRates[key] * j) + reviewRates[key]) / (j + 1);
+};
+
+const numRooms = 100;
+const roomObjs = [];
+
+
+for (let i = 0; i < numRooms; i++) {
```
Comment (tonaramos) [inline database/insertMockData.js]: According to the style guide looks like is better practice to use higher order functions like forEach, map...  to iterate over arrays.

Comment (clfhhc) [reply database/insertMockData.js]: In this case, I don't have an array yet.....

**Pull request closed without merging**